mlapi/model/facet_sense_dto/facet_dto.py

```diff
@@ -0,0 +1,8 @@
+class FacetDto(object):
+    def __init__(self, name, score, values):
+        self.name = name
+        self.score = score
+        self.values = values
+
+    def to_json(self):
```

Comment:
to_json

I'd expect this to return a string or rename to "to_dict" ?

Side note, we don't seem to need a class for the DTO's right now instead of just using dicts. But is it because we expect them to grow and hold more than simple data someday?